android/pom.xml

```diff
@@ -357,6 +358,19 @@
         <maven-javadoc-plugin.additionalJOptions>--no-module-directories</maven-javadoc-plugin.additionalJOptions>
       </properties>
     </profile>
+    <profile>
+      <!--
+          In order to build against JDK 19+, Javadoc needs to enable Java preview features, because JDK sources
+          themselves have code which defines/uses preview features.
+       -->
+      <id>javadocs-jdk19plus</id>
+      <activation>
+        <jdk>[19,]</jdk>
+      </activation>
+      <properties>
+        <maven-javadoc-plugin.additionalJOptions>--enable-preview</maven-javadoc-plugin.additionalJOptions>
+      </properties>
+    </profile>
```

Comment:
On JDK19+, the `--enable-preview` flag is now passed during Javadoc build. This fixes the preview compile failure cited in google/guava#7065.